src/movies.js

```diff
@@ -1,15 +1,88 @@
 /* eslint no-restricted-globals: 'off' */
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
+function orderByYear(movies){
+    movies = JSON.parse((JSON.stringify(movies)))
+     return movies.sort((a, b)=>{
+      if(a['title'] < b['title']){
+        return -1;
+      }
+      if (a['title'] > b['title']){
```

Comment:
you didnt need to do that many condition you could have done something like this:
```
if (a.year===b.year){
            return a.title.localeCompare(b.title);
        }
        return a.year-b.year;
```
That way you could save a lot of code :)